connection.impl/src/main/java/com/ericsson/bss/cassandra/ecchronos/connection/impl/OSGiLocalNativeConnectionProvider.java

```diff
@@ -44,8 +44,11 @@ public class OSGiLocalNativeConnectionProvider implements NativeConnectionProvid
 
     private volatile LocalNativeConnectionProvider myDelegateNativeConnectionProvider;
 
+    /**
```

Comment:
Is this empty comment needed? If yes then put something meaningful

Reply:
Final it is then